### DISCUSSION TITLE
Défaut dans les ressources

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#779] [Q&A]
![image](https://github.com/user-attachments/assets/df03a869-0a0e-421f-a47f-2c6faab87767)

Bonjour,

Souvent j'ai un problème, l'unité ne remonte pas dans le reflet ? 
Il prends l'ancienne unité ? 

Exemple dans la photo

**Comment by manufitoussiwit** [comment]
Bonjour, 

Quelle version de REDY est utilisée ?

**Reply by manufitoussiwit** [reply]
Question supplémentaire : un rafraichissement de la prévisualisation de la scène (fermeture de l'onglet puis réouverture par exemple) permet il de retrouver la bonne unité?

**Reply by FlorentAdell** [reply]
Salut Manu,

Non même dans le Redy il garde no unit 
il est en version 14.5.2

![image](https://github.com/user-attachments/assets/3a4f2ab5-1804-4979-b088-efa784c78542)
